app/views/layouts/_navigation_links.html.erb

```diff
@@ -1,23 +1,39 @@
-<% if session[:line] %>
-  <ul class="nav navbar-nav navbar-left">
-    <li class="navbar-text"><%= current_line_display %></li>
-    <% if ENV['CM_RESOURCES_URL'] %>
-      <li><%= link_to 'CM Resources', ENV['CM_RESOURCES_URL'], target: '_blank' %></li>
-    <% end %>
-  </ul>
-<% end %>
+<ul class="nav navbar-nav navbar-left">
+  <% if current_user && session[:line] %>
+    <li class="navbar-text-alt"><%= current_line_display %></li>
+  <% end %>
+</ul>    
 
 <ul class="nav navbar-nav navbar-right">
   <% if !user_signed_in? %>
     <li><%= link_to 'Sign in', new_user_session_path %></li>
   <% else %>
-    <% unless current_page?(authenticated_root_url) %>
-      <li><%= link_to 'Dashboard', authenticated_root_path %></li>
+    <li><%= link_to 'Reports', '#' %></li>
```

Comment:
In the issues you said there were reports accessible to everyone that you wanted on the top level. Confirming you want to eliminate this?